web/src/app/util/luxon-helpers.js

```diff
@@ -31,3 +31,13 @@ export function getEndOfWeek(luxonDateTime = DateTime.local()) {
     })
     .endOf('day')
 }
+
+// compare two iso datetime strings
+export function isAfter(a, b) {
```

Comment:
Since these are now in a separate file, let's add a little more context/hinting to the names, like `isISOBefore` and `isISOAfter`.

Also, it might be better to have this in `util/shifts.ts`, since these don't actually consume or produce any luxon types. Though honestly, we don't have great organization around utility functions in the first place. :shrug: 

Lastly, let's add proper comments for these

```
// isISOAfter
// Compares two ISO timestamps, returning true if `a` occurs after `b`.
```